libs/api/domains/occupational-licenses/src/lib/models/occupationalLicense.model.ts

```diff
@@ -5,16 +5,37 @@ import {
   InterfaceType,
   registerEnumType,
 } from '@nestjs/graphql'
+import { OccupationalLicensesError } from './occupationalLicenseError.model'
 
 export enum OccupationalLicenseType {
   EDUCATION = 'EDUCATIONAL',
   HEALTH = 'HEALTH_DIRECTORATE',
 }
 
+export enum HealthDirectorateStatusType {
+  valid = 'Í gildi',
+  limited = 'Í gildi - Takmörkun',
+  error = 'Ógilt',
+}
+
+export enum OccupationalLicenseStatus {
+  valid = 'valid',
+  error = 'error',
+  limited = 'limited',
+}
+
```

Comment:
This is almost the same thing, might be worth mapping the `healthDirectorateStatustType` to `OccupationalLicenseStatus`maybe in the Occupational-Licenses domain to make it more generic